lib/babe/state.go

```diff
@@ -33,10 +33,12 @@ type BlockState interface {
 	SubChain(start, end common.Hash) ([]common.Hash, error)
 	AddBlock(*types.Block) error
 	AddBlockWithArrivalTime(*types.Block, uint64) error
-	SetBlock(*types.Block) error
+
+	SetHeader(*types.Header) error
+	GetHeader(common.Hash) (*types.Header, error)
```

Comment:
BABE isn't using these, remove

Reply:
SetHeader is not used, but GetHeader is in use.